src/app/grants/faq/index.tsx

```diff
@@ -0,0 +1,32 @@
+import Item from "./item";
+
+export interface GrantsFaq {
+  answer: string;
+  question: string;
+  lists?: string[];
+}
+
+export default function FaqContainer({ faqs }: { faqs: GrantsFaq[] }) {
+  return (
+    <div className="flex flex-col items-center justify-center gap-10 pt-10 pb-20 lg:pt-[4.5rem] lg:pb-[9rem] px-6">
+      <div className="flex flex-col w-full lg:w-[34.4375rem] gap-4 items-center justify-center">
+        <h3 className="text-base font-poppins font-medium leading-[2rem] text-[#C6C6C6] text-center">
+          FAQs
+        </h3>
+        <h4 className="font-poppins font-bold text-[1.5rem] leading-[2.25rem] lg:text-[2rem] lg:leading-[3rem] text-center text-[#FDFDFD]">
+          Have Questions about this Grant? We have answers
```

Comment:
Given this is the main page not for a specific grant:

```suggestion
          Have Questions about our Grants? We have answers
```